Character restrictions are not enforced via the commandline

<!-- Please use this template while reporting a bug and provide as much info as possible. Not doing so may result in your bug not being addressed in a timely manner. Thanks!

If the matter is security related, please disclose it privately via https://github.com/nventive/Uno/security/
-->

## Current behavior

<!-- Describe how the issue manifests. -->


When creating a project from the templates in Visual Studio, there is a check for which characters are acceptable as part of the project (or solution) name


There's no equivalent test for the project name when creating a solution from the command line.
Is this an oversight? 


This means it's possible to create apps that won't build, and there's no obvious indication of what's wrong from the error messages that are produced.
I understand that limiting the characters accepted avoids the need to deal with safe project names and escape characters, but I wanted to draw attention to the fact that these custom rules aren't being applied consistently when projects are being created outside of VS. 


## Expected behavior

<!-- Describe what the desired behavior would be. -->

Characters that are valid as part of a project name can be used, regardless of whether the wizard is used or the templates are used from the command line.

## How to reproduce it (as minimally and precisely as possible)

<!-- Please provide a **MINIMAL REPRO PROJECT** and the **STEPS TO REPRODUCE**-->

## Environment

<!-- For bug reports Check one or more of the following options with "x" -->

Nuget Package:

Package Version(s):

Affected platform(s):

- [ ] iOS
- [ ] Android
- [ ] WebAssembly
- [ ] WebAssembly renders for Xamarin.Forms
- [ ] Windows
- [ ] Build tasks

Visual Studio:

- [ ] 2017 (version: )
- [ ] 2019 (version: )
- [ ] for Mac (version: )

Relevant plugins:

- [ ] Resharper (version: )

## Anything else we need to know?

<!-- We would love to know of any friction, apart from knowledge, that prevented you from sending in a pull-request -->

The equals sign is technically valid as part of a project name but currently breaks on .net 8 due to https://github.com/dotnet/roslyn/issues/71022

example errors:

```ascii
PS C:\UIT\brackets(are)around> dotnet build
MSBuild version 17.8.3+195e7f5a3 for .NET
  Determining projects to restore...
  All projects are up-to-date for restore.
C:\UIT\brackets(are)around\brackets(are)around\Uno.UI.SourceGenerators\Uno.UI.SourceGenerators.XamlGenerator.XamlCodeGe
nerator\MainPage_6a617185d6ea16dfe5a4f90da3b777e4.cs(27,15): error CS8802: Only one compilation unit can have top-level
 statements. [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0]
C:\UIT\brackets(are)around\brackets(are)around\Uno.UI.SourceGenerators\Uno.UI.SourceGenerators.XamlGenerator.XamlCodeGe
nerator\MainPage_6a617185d6ea16dfe5a4f90da3b777e4.cs(27,15): error CS8802: Only one compilation unit can have top-level
 statements. [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-ios]
C:\UIT\brackets(are)around\brackets(are)around\Uno.UI.SourceGenerators\Uno.UI.SourceGenerators.XamlGenerator.XamlCodeGe
nerator\MainPage_6a617185d6ea16dfe5a4f90da3b777e4.cs(27,15): error CS8802: Only one compilation unit can have top-level
 statements. [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-maccata
lyst]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-android\__Microsoft.Android.Resource.Designer.cs(8,20):
 error CS0116: A namespace cannot directly contain members such as fields, methods or statements [C:\UIT\brackets(are)a
round\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-android]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-android\__Microsoft.Android.Resource.Designer.cs(8,24):
 error CS0116: A namespace cannot directly contain members such as fields, methods or statements [C:\UIT\brackets(are)a
round\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-android]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-android\__Microsoft.Android.Resource.Designer.cs(8,19):
 error CS1514: { expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=n
et8.0-android]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-android\__Microsoft.Android.Resource.Designer.cs(8,23):
 error CS1022: Type or namespace definition, or end-of-file expected [C:\UIT\brackets(are)around\brackets(are)around\br
ackets(are)around.csproj::TargetFramework=net8.0-android]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-android\__Microsoft.Android.Resource.Designer.cs(8,31):
 error CS1022: Type or namespace definition, or end-of-file expected [C:\UIT\brackets(are)around\brackets(are)around\br
ackets(are)around.csproj::TargetFramework=net8.0-android]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(12,20): error CS011
6: A namespace cannot directly contain members such as fields, methods or statements [C:\UIT\brackets(are)around\bracke
ts(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(12,31): error CS011
6: A namespace cannot directly contain members such as fields, methods or statements [C:\UIT\brackets(are)around\bracke
ts(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(12,19): error CS151
4: { expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-window
s10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(12,23): error CS102
2: Type or namespace definition, or end-of-file expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)a
round.csproj::TargetFramework=net8.0-windows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(13,1): error CS1022
: Type or namespace definition, or end-of-file expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)ar
ound.csproj::TargetFramework=net8.0-windows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(21,33): error CS151
9: Invalid token '(' in class, record, struct, or interface member declaration [C:\UIT\brackets(are)around\brackets(are
)around\brackets(are)around.csproj::TargetFramework=net8.0-windows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(21,37): error CS812
4: Tuple must contain at least two elements. [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj
::TargetFramework=net8.0-windows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(21,99): error CS100
3: Syntax error, '(' expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramewo
rk=net8.0-windows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(21,109): error CS10
01: Identifier expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net
8.0-windows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(21,115): error CS10
26: ) expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(23,33): error CS151
9: Invalid token '(' in class, record, struct, or interface member declaration [C:\UIT\brackets(are)around\brackets(are
)around\brackets(are)around.csproj::TargetFramework=net8.0-windows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(23,37): error CS812
4: Tuple must contain at least two elements. [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj
::TargetFramework=net8.0-windows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(23,99): error CS100
3: Syntax error, '(' expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramewo
rk=net8.0-windows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(23,107): error CS10
01: Identifier expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net
8.0-windows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(23,107): error CS10
26: ) expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(25,16): error CS100
2: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-window
s10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(29,58): error CS100
2: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-window
s10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(79,64): error CS100
2: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-window
s10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(118,64): error CS10
02: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(253,34): error CS10
02: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(254,34): error CS10
02: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(262,53): error CS10
02: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(270,53): error CS10
02: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(274,53): error CS10
02: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(278,53): error CS10
02: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(282,53): error CS10
02: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(289,53): error CS10
02: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(293,53): error CS10
02: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(297,53): error CS10
02: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(311,53): error CS10
02: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(315,53): error CS10
02: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(322,53): error CS10
02: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(448,34): error CS10
02: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(449,34): error CS10
02: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(454,50): error CS10
26: ) expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(454,50): error CS10
02: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(454,102): error CS1
002: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-wind
ows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(454,102): error CS1
513: } expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-wind
ows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(455,55): error CS10
02: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(461,50): error CS10
26: ) expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(461,50): error CS10
02: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(461,102): error CS1
002: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-wind
ows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(461,102): error CS1
513: } expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-wind
ows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(462,55): error CS10
02: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(467,50): error CS10
26: ) expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(467,50): error CS10
02: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(467,102): error CS1
002: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-wind
ows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(467,102): error CS1
513: } expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-wind
ows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(468,55): error CS10
02: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(474,50): error CS10
26: ) expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(474,50): error CS10
02: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(474,102): error CS1
002: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-wind
ows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(474,102): error CS1
513: } expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-wind
ows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(475,55): error CS10
02: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(481,50): error CS10
26: ) expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(481,50): error CS10
02: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(481,102): error CS1
002: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-wind
ows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(481,102): error CS1
513: } expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-wind
ows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(482,55): error CS10
02: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(488,50): error CS10
26: ) expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(488,50): error CS10
02: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(488,102): error CS1
002: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-wind
ows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(488,102): error CS1
513: } expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-wind
ows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(489,55): error CS10
02: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(494,50): error CS10
26: ) expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(494,50): error CS10
02: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(494,102): error CS1
002: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-wind
ows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(494,102): error CS1
513: } expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-wind
ows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(495,55): error CS10
02: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(501,50): error CS10
26: ) expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(501,50): error CS10
02: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(501,102): error CS1
002: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-wind
ows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(501,102): error CS1
513: } expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-wind
ows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(502,55): error CS10
02: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(563,56): error CS10
03: Syntax error, ',' expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramew
ork=net8.0-windows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(566,25): error CS15
19: Invalid token '(' in class, record, struct, or interface member declaration [C:\UIT\brackets(are)around\brackets(ar
e)around\brackets(are)around.csproj::TargetFramework=net8.0-windows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(566,29): error CS81
24: Tuple must contain at least two elements. [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.cspro
j::TargetFramework=net8.0-windows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(566,91): error CS10
03: Syntax error, '(' expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramew
ork=net8.0-windows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(566,100): error CS1
001: Identifier expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=ne
t8.0-windows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(566,100): error CS1
026: ) expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-wind
ows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(581,45): error CS10
01: Identifier expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net
8.0-windows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(581,45): error CS10
03: Syntax error, ',' expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramew
ork=net8.0-windows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(581,49): error CS81
24: Tuple must contain at least two elements. [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.cspro
j::TargetFramework=net8.0-windows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(581,56): error CS10
03: Syntax error, ',' expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramew
ork=net8.0-windows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(581,57): error CS10
03: Syntax error, ',' expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramew
ork=net8.0-windows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(796,25): error CS15
19: Invalid token '(' in class, record, struct, or interface member declaration [C:\UIT\brackets(are)around\brackets(ar
e)around\brackets(are)around.csproj::TargetFramework=net8.0-windows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(796,29): error CS81
24: Tuple must contain at least two elements. [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.cspro
j::TargetFramework=net8.0-windows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(796,91): error CS10
03: Syntax error, '(' expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramew
ork=net8.0-windows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(796,100): error CS1
001: Identifier expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=ne
t8.0-windows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(796,100): error CS1
026: ) expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-wind
ows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(805,43): error CS10
01: Identifier expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net
8.0-windows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(805,43): error CS10
03: Syntax error, ',' expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramew
ork=net8.0-windows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(805,47): error CS81
24: Tuple must contain at least two elements. [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.cspro
j::TargetFramework=net8.0-windows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(805,54): error CS10
03: Syntax error, ',' expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramew
ork=net8.0-windows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(805,55): error CS10
03: Syntax error, ',' expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramew
ork=net8.0-windows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around.Mobile\obj\Debug\net8.0-android\__Microsoft.Android.Resource.Designer.cs
(8,20): error CS0116: A namespace cannot directly contain members such as fields, methods or statements [C:\UIT\bracket
s(are)around\brackets(are)around.Mobile\brackets(are)around.Mobile.csproj::TargetFramework=net8.0-android]
C:\UIT\brackets(are)around\brackets(are)around.Mobile\obj\Debug\net8.0-android\__Microsoft.Android.Resource.Designer.cs
(8,31): error CS0116: A namespace cannot directly contain members such as fields, methods or statements [C:\UIT\bracket
s(are)around\brackets(are)around.Mobile\brackets(are)around.Mobile.csproj::TargetFramework=net8.0-android]
C:\UIT\brackets(are)around\brackets(are)around.Mobile\obj\Debug\net8.0-android\__Microsoft.Android.Resource.Designer.cs
(8,19): error CS1514: { expected [C:\UIT\brackets(are)around\brackets(are)around.Mobile\brackets(are)around.Mobile.cspr
oj::TargetFramework=net8.0-android]
C:\UIT\brackets(are)around\brackets(are)around.Mobile\obj\Debug\net8.0-android\__Microsoft.Android.Resource.Designer.cs
(8,23): error CS1022: Type or namespace definition, or end-of-file expected [C:\UIT\brackets(are)around\brackets(are)ar
ound.Mobile\brackets(are)around.Mobile.csproj::TargetFramework=net8.0-android]
C:\UIT\brackets(are)around\brackets(are)around.Mobile\obj\Debug\net8.0-android\__Microsoft.Android.Resource.Designer.cs
(8,38): error CS1022: Type or namespace definition, or end-of-file expected [C:\UIT\brackets(are)around\brackets(are)ar
ound.Mobile\brackets(are)around.Mobile.csproj::TargetFramework=net8.0-android]

Build FAILED.

C:\UIT\brackets(are)around\brackets(are)around\Uno.UI.SourceGenerators\Uno.UI.SourceGenerators.XamlGenerator.XamlCodeGe
nerator\MainPage_6a617185d6ea16dfe5a4f90da3b777e4.cs(27,15): error CS8802: Only one compilation unit can have top-level
 statements. [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0]
C:\UIT\brackets(are)around\brackets(are)around\Uno.UI.SourceGenerators\Uno.UI.SourceGenerators.XamlGenerator.XamlCodeGe
nerator\MainPage_6a617185d6ea16dfe5a4f90da3b777e4.cs(27,15): error CS8802: Only one compilation unit can have top-level
 statements. [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-ios]
C:\UIT\brackets(are)around\brackets(are)around\Uno.UI.SourceGenerators\Uno.UI.SourceGenerators.XamlGenerator.XamlCodeGe
nerator\MainPage_6a617185d6ea16dfe5a4f90da3b777e4.cs(27,15): error CS8802: Only one compilation unit can have top-level
 statements. [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-maccata
lyst]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-android\__Microsoft.Android.Resource.Designer.cs(8,20):
 error CS0116: A namespace cannot directly contain members such as fields, methods or statements [C:\UIT\brackets(are)a
round\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-android]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-android\__Microsoft.Android.Resource.Designer.cs(8,24):
 error CS0116: A namespace cannot directly contain members such as fields, methods or statements [C:\UIT\brackets(are)a
round\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-android]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-android\__Microsoft.Android.Resource.Designer.cs(8,19):
 error CS1514: { expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=n
et8.0-android]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-android\__Microsoft.Android.Resource.Designer.cs(8,23):
 error CS1022: Type or namespace definition, or end-of-file expected [C:\UIT\brackets(are)around\brackets(are)around\br
ackets(are)around.csproj::TargetFramework=net8.0-android]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-android\__Microsoft.Android.Resource.Designer.cs(8,31):
 error CS1022: Type or namespace definition, or end-of-file expected [C:\UIT\brackets(are)around\brackets(are)around\br
ackets(are)around.csproj::TargetFramework=net8.0-android]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(12,20): error CS011
6: A namespace cannot directly contain members such as fields, methods or statements [C:\UIT\brackets(are)around\bracke
ts(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(12,31): error CS011
6: A namespace cannot directly contain members such as fields, methods or statements [C:\UIT\brackets(are)around\bracke
ts(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(12,19): error CS151
4: { expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-window
s10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(12,23): error CS102
2: Type or namespace definition, or end-of-file expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)a
round.csproj::TargetFramework=net8.0-windows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(13,1): error CS1022
: Type or namespace definition, or end-of-file expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)ar
ound.csproj::TargetFramework=net8.0-windows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(21,33): error CS151
9: Invalid token '(' in class, record, struct, or interface member declaration [C:\UIT\brackets(are)around\brackets(are
)around\brackets(are)around.csproj::TargetFramework=net8.0-windows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(21,37): error CS812
4: Tuple must contain at least two elements. [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj
::TargetFramework=net8.0-windows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(21,99): error CS100
3: Syntax error, '(' expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramewo
rk=net8.0-windows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(21,109): error CS10
01: Identifier expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net
8.0-windows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(21,115): error CS10
26: ) expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(23,33): error CS151
9: Invalid token '(' in class, record, struct, or interface member declaration [C:\UIT\brackets(are)around\brackets(are
)around\brackets(are)around.csproj::TargetFramework=net8.0-windows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(23,37): error CS812
4: Tuple must contain at least two elements. [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj
::TargetFramework=net8.0-windows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(23,99): error CS100
3: Syntax error, '(' expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramewo
rk=net8.0-windows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(23,107): error CS10
01: Identifier expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net
8.0-windows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(23,107): error CS10
26: ) expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(25,16): error CS100
2: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-window
s10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(29,58): error CS100
2: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-window
s10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(79,64): error CS100
2: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-window
s10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(118,64): error CS10
02: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(253,34): error CS10
02: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(254,34): error CS10
02: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(262,53): error CS10
02: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(270,53): error CS10
02: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(274,53): error CS10
02: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(278,53): error CS10
02: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(282,53): error CS10
02: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(289,53): error CS10
02: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(293,53): error CS10
02: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(297,53): error CS10
02: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(311,53): error CS10
02: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(315,53): error CS10
02: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(322,53): error CS10
02: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(448,34): error CS10
02: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(449,34): error CS10
02: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(454,50): error CS10
26: ) expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(454,50): error CS10
02: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(454,102): error CS1
002: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-wind
ows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(454,102): error CS1
513: } expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-wind
ows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(455,55): error CS10
02: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(461,50): error CS10
26: ) expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(461,50): error CS10
02: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(461,102): error CS1
002: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-wind
ows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(461,102): error CS1
513: } expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-wind
ows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(462,55): error CS10
02: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(467,50): error CS10
26: ) expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(467,50): error CS10
02: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(467,102): error CS1
002: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-wind
ows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(467,102): error CS1
513: } expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-wind
ows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(468,55): error CS10
02: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(474,50): error CS10
26: ) expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(474,50): error CS10
02: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(474,102): error CS1
002: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-wind
ows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(474,102): error CS1
513: } expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-wind
ows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(475,55): error CS10
02: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(481,50): error CS10
26: ) expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(481,50): error CS10
02: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(481,102): error CS1
002: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-wind
ows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(481,102): error CS1
513: } expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-wind
ows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(482,55): error CS10
02: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(488,50): error CS10
26: ) expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(488,50): error CS10
02: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(488,102): error CS1
002: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-wind
ows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(488,102): error CS1
513: } expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-wind
ows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(489,55): error CS10
02: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(494,50): error CS10
26: ) expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(494,50): error CS10
02: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(494,102): error CS1
002: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-wind
ows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(494,102): error CS1
513: } expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-wind
ows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(495,55): error CS10
02: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(501,50): error CS10
26: ) expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(501,50): error CS10
02: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(501,102): error CS1
002: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-wind
ows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(501,102): error CS1
513: } expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-wind
ows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(502,55): error CS10
02: ; expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-windo
ws10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(563,56): error CS10
03: Syntax error, ',' expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramew
ork=net8.0-windows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(566,25): error CS15
19: Invalid token '(' in class, record, struct, or interface member declaration [C:\UIT\brackets(are)around\brackets(ar
e)around\brackets(are)around.csproj::TargetFramework=net8.0-windows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(566,29): error CS81
24: Tuple must contain at least two elements. [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.cspro
j::TargetFramework=net8.0-windows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(566,91): error CS10
03: Syntax error, '(' expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramew
ork=net8.0-windows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(566,100): error CS1
001: Identifier expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=ne
t8.0-windows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(566,100): error CS1
026: ) expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-wind
ows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(581,45): error CS10
01: Identifier expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net
8.0-windows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(581,45): error CS10
03: Syntax error, ',' expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramew
ork=net8.0-windows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(581,49): error CS81
24: Tuple must contain at least two elements. [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.cspro
j::TargetFramework=net8.0-windows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(581,56): error CS10
03: Syntax error, ',' expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramew
ork=net8.0-windows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(581,57): error CS10
03: Syntax error, ',' expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramew
ork=net8.0-windows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(796,25): error CS15
19: Invalid token '(' in class, record, struct, or interface member declaration [C:\UIT\brackets(are)around\brackets(ar
e)around\brackets(are)around.csproj::TargetFramework=net8.0-windows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(796,29): error CS81
24: Tuple must contain at least two elements. [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.cspro
j::TargetFramework=net8.0-windows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(796,91): error CS10
03: Syntax error, '(' expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramew
ork=net8.0-windows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(796,100): error CS1
001: Identifier expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=ne
t8.0-windows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(796,100): error CS1
026: ) expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net8.0-wind
ows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(805,43): error CS10
01: Identifier expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramework=net
8.0-windows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(805,43): error CS10
03: Syntax error, ',' expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramew
ork=net8.0-windows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(805,47): error CS81
24: Tuple must contain at least two elements. [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.cspro
j::TargetFramework=net8.0-windows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(805,54): error CS10
03: Syntax error, ',' expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramew
ork=net8.0-windows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around\obj\Debug\net8.0-windows10.0.19041\XamlTypeInfo.g.cs(805,55): error CS10
03: Syntax error, ',' expected [C:\UIT\brackets(are)around\brackets(are)around\brackets(are)around.csproj::TargetFramew
ork=net8.0-windows10.0.19041]
C:\UIT\brackets(are)around\brackets(are)around.Mobile\obj\Debug\net8.0-android\__Microsoft.Android.Resource.Designer.cs
(8,20): error CS0116: A namespace cannot directly contain members such as fields, methods or statements [C:\UIT\bracket
s(are)around\brackets(are)around.Mobile\brackets(are)around.Mobile.csproj::TargetFramework=net8.0-android]
C:\UIT\brackets(are)around\brackets(are)around.Mobile\obj\Debug\net8.0-android\__Microsoft.Android.Resource.Designer.cs
(8,31): error CS0116: A namespace cannot directly contain members such as fields, methods or statements [C:\UIT\bracket
s(are)around\brackets(are)around.Mobile\brackets(are)around.Mobile.csproj::TargetFramework=net8.0-android]
C:\UIT\brackets(are)around\brackets(are)around.Mobile\obj\Debug\net8.0-android\__Microsoft.Android.Resource.Designer.cs
(8,19): error CS1514: { expected [C:\UIT\brackets(are)around\brackets(are)around.Mobile\brackets(are)around.Mobile.cspr
oj::TargetFramework=net8.0-android]
C:\UIT\brackets(are)around\brackets(are)around.Mobile\obj\Debug\net8.0-android\__Microsoft.Android.Resource.Designer.cs
(8,23): error CS1022: Type or namespace definition, or end-of-file expected [C:\UIT\brackets(are)around\brackets(are)ar
ound.Mobile\brackets(are)around.Mobile.csproj::TargetFramework=net8.0-android]
C:\UIT\brackets(are)around\brackets(are)around.Mobile\obj\Debug\net8.0-android\__Microsoft.Android.Resource.Designer.cs
(8,38): error CS1022: Type or namespace definition, or end-of-file expected [C:\UIT\brackets(are)around\brackets(are)ar
ound.Mobile\brackets(are)around.Mobile.csproj::TargetFramework=net8.0-android]
    0 Warning(s)
    108 Error(s)

Time Elapsed 00:00:14.65
```

--- a/src/Uno.Templates/content/unoapp/.template.config/template.json
+++ b/src/Uno.Templates/content/unoapp/.template.config/template.json
@@ -83,6 +83,15 @@
       },
       "replaces": "$sanitizedProjectName$"
     },
+    "failBuildOnInvalidProjectName": {
+      "type": "generated",
+      "generator": "regexMatch",
+      "datatype": "bool",
+      "parameters": {
+        "source": "name",
+        "pattern": "\\s"
+      }
+    },
     "appId": {
       "displayName": "Application ID",
       "type": "parameter",

--- a/src/Uno.Templates/content/unoapp/Directory.Build.targets
+++ b/src/Uno.Templates/content/unoapp/Directory.Build.targets
@@ -1,3 +1,8 @@
 ﻿<Project>
-
+<!--#if (failBuildOnInvalidProjectName)-->
+  <Target Name="InvalidProjectName"
+          BeforeTargets="BeforeBuild">
+    <Error Text="The project name 'MyExtensionsApp.1' is invalid. Your project name may not contain spaces as this will break various .NET Source Generators and MSBuild targets." />
+  </Target>
+<!--#endif-->
 </Project>

